src/Services/PatientService.php

```diff
@@ -200,6 +206,10 @@ public function update($pid, $data)
         );
     }
 
+    public function insertById($pid,$data){
+        return updatePatientData($pid,$data,'true');
+    }
```

Comment:
isn't this means that the user wants to create on particular PID? if user wants to create a user by certain user id then I think it will be required to pass PID

Reply:
We don't want to allow create from a PUT! That's what insert is for plus pid is created from a sequence.
Also our patient insert should also be creating a blank insurance record. Just like a create patient workflow...